docs/src/references/api/torch/systems.rst

```diff
@@ -1,7 +1,4 @@
 System
 ======
 
-.. autoclass:: rascaline.torch.System
```

Comment:
Maybe add a comment that we are using the metatensor systems class and we provide some conversion functions.

Reply:
done!